Adyen/CheckoutConfiguration/CheckoutTheme.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// `CheckoutTheme` represents theme of the SDK required for dropIn and components UI.
+public protocol CheckoutTheme {
```

Comment:
I see this protocol and below is added to allow access from the UI module, but does it need to be public? merchants shouldn't need to know about the protocols, just the actual theme classes

Reply:
I have change it to `package`